Langs/080C.lang

```diff
@@ -0,0 +1,79 @@
+[MetaData]
+Language=French - Belgium
+Translator=PierreGrillet; Nits77; villisek; BSoD38
```

Comment:
Is it needed to add all translators to this file? What if they don't know that version of French and there are some subtle differences?

Reply:
Frankly I just copy paste the French France translation (and their authors), I don't think they will be such difference, as translation can already been discussed. (Should we translate that or not ? Is that not to far fetch ? etc...)

The french spoken in the bordering countries is very close anyway, so in a technical environment like computing, I don't think they is much difference.

However, for Quebec French some major difference can exist, as they are more strict and translate more terms (in general) (Never tell a Quebecer that he live in Canada :wink:, that the best way to not understand them).

My best bet, is to wait for an review by a Quebec french speaker. Maybe for other.

Asking review from any french speaker, or from @Nits77, @villisek, @vincent64. :+1:  :smile: